test/data/bad_notices/notice_type_as_attr.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I see two .json for test the "bad condition" in VR API. What abotu a json to test it in the plain rules API?